### DISCUSSION TITLE
[Application] Fildata.info

### DISCUSSION THREAD

**Discussion by daichi000** [filecoin-project/community#235] [Project Showcase]
## Project Description
[FilData](https://www.fildata.info/en) is a Filecoin network browser, which is very useful to get chain data of Filecoin network and do data analysis.

## Project Status
FilData has released the first version, which has completed basic functions and can meet the most people's needs. More functions are beening developed, we will open an discussion in the GitHub, hope to get more useful suggestions.

## Previews
![image](https://user-images.githubusercontent.com/35623457/127434339-fbbe3bd2-9c90-458d-a3a0-907b5fe1fac7.png)
![image](https://user-images.githubusercontent.com/35623457/127434485-92c4ef62-8b61-4db5-9901-f8bd3afdc967.png)
![image](https://user-images.githubusercontent.com/35623457/127434932-28593f55-a886-4b1f-a9e4-0cd6130d2bef.png)

## Target Audience
Everybody can use it to get the Filecoin Network chain infomations. Miners can get the relative mining infomation, clients can get the miners' status and chain status and so on.

**Comment by longfeiWan9** [comment]
@daichi000 Nice job. 
Since you mention it will be very useful to get chain data of Filecoin network and do data analysis, I am wondering will it provide some APIs for getting chain data and mining information for developers or storage providers ?
